src/main/java/kitchenpos/products/tobe/domain/DisplayedName.java

```diff
@@ -0,0 +1,54 @@
+package kitchenpos.products.tobe.domain;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class DisplayedName {
+
+  @Column(name = "name", nullable = false)
+  private String name;
+
+  public static DisplayedName from(String name) {
+    return new DisplayedName(name);
+  }
+
+  public static DisplayedName from(String name, DisplayNameValidator validator) {
+    DisplayedName displayedName = new DisplayedName(name);
+    validator.validate(displayedName);
+    return displayedName;
+  }
+
+  public DisplayedName() {
```

Comment:
접근제어자는 필요한 만큼만 노출하는 것은 어떤가요?
```suggestion
  protected DisplayedName() {
```

Reply:
아 넵 인텔리제이 자동생성 기능으로 습관처럼 만들다보니 미처 생각 못 했네요!
좋은 피드백 감사합니다 👍